ts/webui/src/components/managementExp/Experiment.tsx

```diff
@@ -0,0 +1,433 @@
+                                    selectPlatform={this.selectPlatform.bind(this)}
+                                    getSelectedData={this.getSelectedData.bind(this)}
+                                    setSearchSource={this.setSearchSource.bind(this)}
+                                />
+                            </Stack>
+                            <DetailsList
+                                columns={this.columns}
+                                items={source}
+                                setKey='set'
+                                compact={true}
+                                selectionMode={0} // close selector function
+                                className='table'
+                            />
+                        </Stack>
+                    </Stack>
+                </Stack>
+            </Stack>
+        );
+    }
+
+    private onColumnClick = (_ev: React.MouseEvent<HTMLElement>, getColumn: IColumn): void => {
+        const { columns, source } = this.state;
+        const newColumns: IColumn[] = columns.slice();
+        const currColumn: IColumn = newColumns.filter(item => getColumn.key === item.key)[0];
+        newColumns.forEach((newCol: IColumn) => {
+            if (newCol === currColumn) {
+                currColumn.isSortedDescending = !currColumn.isSortedDescending;
+                currColumn.isSorted = true;
+            } else {
+                newCol.isSorted = false;
+                newCol.isSortedDescending = true;
+            }
+        });
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        const newItems = copyAndSort(source, currColumn.fieldName!, currColumn.isSortedDescending);
+        this.setState(() => ({
+            columns: newColumns,
+            source: newItems,
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            sortInfo: { field: currColumn.fieldName!, isDescend: currColumn.isSortedDescending }
+        }));
+    };
+
+    private columns: IColumn[] = [
+        {
+            name: 'Name',
+            key: 'experimentName',
+            fieldName: 'experimentName', // required!
+            minWidth: MINSCREENCOLUMNWIDHT,
+            maxWidth: MAXSCREENCOLUMNWIDHT,
+            isResizable: true,
+            data: 'number',
+            onColumnClick: this.onColumnClick,
+            onRender: (item: any): React.ReactNode => <NameColumn port={item.port} expName={item.experimentName} />
+        },
+        {
+            name: 'ID',
+            key: 'id',
+            fieldName: 'id',
+            minWidth: MINSCREENCOLUMNWIDHT,
+            maxWidth: MAXSCREENCOLUMNWIDHT,
+            isResizable: true,
+            className: 'tableHead leftTitle',
+            data: 'string',
+            onColumnClick: this.onColumnClick,
+            onRender: (item: any): React.ReactNode => <div className='succeed-padding'>{item.id}</div>
```

Comment:
Suggest to make "ID" and "Status" clickable.
And personally suggest to remove underscore from "Name".

Reply:
discuss with xingyu, and now `ID`  column is clickable.